Cargo.toml

```diff
@@ -32,6 +32,7 @@ quote = { version = "1.0.23" }
 regex = { version = "1.7.1" }
 rustc-hash = { version = "1.1.0" }
 ruff_text_size = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "335780aeeac1e6fcd85994ba001d7b8ce99fcf65" }
+rustpython-ast = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "335780aeeac1e6fcd85994ba001d7b8ce99fcf65", default-features = false, features = ["all-nodes-with-ranges"]}
```

Comment:
Do we not actually depend on this anywhere?

Reply:
I know started using it in `ruff_python_ast` because our AST crate would, ideally, only depend on `ruff_python_ast` and not the lexer/parser.